### PR TITLE
Trust custom certificates

### PR DESCRIPTION
**Pull request by tschanfeld**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34417

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by fvigo** [comment]
@tschanfeld is it ready for review? If so please assign a reviewer! Thanks!

File: content-repo/extra-docs/articles/trust-custom-certificates.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- "/etc/ssl/ca-bundle.pem", // OpenSUSE
+- "/etc/pki/tls/cacert.pem", // OpenELEC
+- "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem", // CentOS/RHEL 7
+- "/etc/ssl/cert.pem", // Alpine Linux
+
+Examples of certificate bundle directories:
+- "/etc/ssl/certs", // SLES10/SLES11, https://golang.org/issue/12139
+- "/etc/pki/tls/certs", // Fedora/RHEL
+
+2. Reboot the server.
```
Comment (glicht) [inline content-repo/extra-docs/articles/trust-custom-certificates.md:22]: Better to use restart. If you have a link with instructions, would be nice to add this here too.
```suggestion
2. Restart the Server.
```

**Review by glicht — CHANGES_REQUESTED** [review]
Please add a note mentioning that for Python integrations there is need to follow the instructions at: https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-1/cortex-xsoar-admin/docker/configure-python-docker-integrations-to-trust-custom-certificates.html

File: content-repo/extra-docs/articles/trust-custom-certificates.md (hunk 1 above)
Comment (glicht) [reply content-repo/extra-docs/articles/trust-custom-certificates.md:22]: Also mention that the Engine requires a restart after this configuration change.

**Comment by glicht** [comment]
@tschanfeld can you go over the formatting. The bullets don't look good:

image

**Review by tschanfeld — COMMENTED** [review]
@glicht Is this approved (removed bulleted list)?

**Review by glicht — APPROVED** [review]

**Comment by kirbles19** [comment]
@glicht Done with the review, ready to merge

**Review by kirbles19 — APPROVED** [review]

**Comment by tschanfeld** [comment]
@kirbles19  Can someone squash/merge if this is ready to go?

**Comment by fvigo** [comment]
@tschanfeld done

**Pull request merged**